### ISSUE TITLE
Exclude ardulink-core-virtual from assembly

### ISSUE THREAD

**Issue by pfichtner** [Ardulink/Ardulink-2#13]
ardulink-core-virtual should not get distributed so it should be excluded in assembly.xml

**Comment by LucianoZu** [comment]
Why? If we exclude it from assembly then we have to remove it from ardulink-console pom.xml

**Comment by pfichtner** [comment]
-Who did add it to ardulink-console pom.xml?
-Why was it added there?
-If so: For the moment the poms do not have any impact on the dist zip, do they?
-All the runtime deps in ardulink-console pom.xml should be marked as optional
-Why shall we distribute ardulink-core-virtual? It was intended for testing purposes

**Comment by pfichtner** [comment]
I see, it was added by you. My intention was to not add it.

**Comment by pfichtner** [comment]
I did mark the runtime deps of ardulink-console as optional

**Comment by LucianoZu** [comment]
Yes, I added to try it with swing components. It work very well and I
supposed that is not a great problem add it to distribution. We can remove
it from pom (or mark it as optional) without problem.

We are a two members... so if a thing is not done from you... 😉

2016-05-14 20:33 GMT+02:00 Peter Fichtner <notifications@github.com>:

> I did mark the runtime deps of ardulink-console as optional
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/Ardulink/Ardulink-2/issues/13#issuecomment-219235960>
>



-- 

   - Profilo LinkedIn: http://www.linkedin.com/in/lucianozu
   - Ardulink: http://www.ardulink.org/
   - Lucio Caio Autore: http://luciocaio.altervista.org/

**Comment by pfichtner** [comment]
If we don't exclude it in assembly.xml it will be distributed. Do you want to distribute it? Distribution means it will be available in console's drop down combo. In my opinion this does not make any sense.

**Comment by LucianoZu** [comment]
No, I agree with you.

2016-05-14 21:32 GMT+02:00 Peter Fichtner <notifications@github.com>:

> If we don't exclude it in assembly.xml it will be distributed. Do you want
> to distribute it? Distribution means it will be available in console's drop
> down combo. In my opinion this does not make any sense.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/Ardulink/Ardulink-2/issues/13#issuecomment-219238849>
>



-- 

   - Profilo LinkedIn: http://www.linkedin.com/in/lucianozu
   - Ardulink: http://www.ardulink.org/
   - Lucio Caio Autore: http://luciocaio.altervista.org/
